src/app/app.component.css

```diff
@@ -0,0 +1,21 @@
+/* AppComponent's private CSS styles */
```

Comment:
Don't think you need to explain what these files are for as it should be inferred by the folder and filename